requirements.txt

```diff
@@ -1,56 +1,56 @@
-argon2-cffi==19.1.0
-autobahn==19.6.2
+argon2-cffi==19.2.0
+autobahn==19.11.1
 Babel==2.6.0
 bcrypt==3.1.7
-beautifulsoup4==4.7.1
+beautifulsoup4==4.8.1
 bleach==3.1.0
 celery==4.3.0
-certifi==2019.06.16
-channels==2.2.0
-channels-redis==2.4.0
-contextlib2==0.5.5
-cryptography==2.7
-decorator==4.3.0
-Django==1.11.23
-django-cacheops==4.1
-django-cors-headers==3.0.2
+certifi==2019.09.11
+channels==2.3.1
+channels-redis==2.4.1
+contextlib2==0.6.0
+cryptography==2.8
+decorator==4.3.2
+Django==1.11.26
+django-cacheops==4.2
+django-cors-headers==3.1.1
 django-debug-toolbar==2.0
-django-extensions==2.1.9
+django-extensions==2.2.5
 django-formtools==2.1
 django-inline-svg==0.1.1
 django-maintenance-mode==0.13.3
-django-oauth-toolkit==1.1.2
+django-oauth-toolkit==1.1.3
 django-pipeline==1.6.14
-django-prometheus==1.0.15
+django-prometheus==1.1.0
 django-redis-cache==2.0.0
 django-redis-sessions==0.6.1
+django-referrer-policy==1.0
 django-requestlogging-redux==1.2.1
-django-simple-history==2.7.2
+django-simple-history==2.7.3
 django-user-agents==0.4.0
 django-widget-tweaks==1.4.5
-djangorestframework==3.9.4
-docutils==0.14
+djangorestframework==3.10.3
+docutils==0.15.2
 Fabric3==1.14.post1
 flower==0.9.3
 gunicorn==19.9.0
 hiredis==1.0.0
-ipython==7.6.1
+ipython==7.9.0
 objgraph==3.4.1
 pexpect==4.7.0
-psycopg2==2.8.3
-pycryptodome==3.8.2
+psycopg2==2.8.4
+pycryptodome==3.9.1
 pysftp==0.2.9
-python-dateutil==2.8.0
+python-dateutil==2.8.1
 python-gssapi==0.6.4
 python-magic==0.4.15
-reportlab==3.5.23
+reportlab==3.5.26
 requests==2.22.0
-requests-oauthlib==1.2.0
-sentry-sdk==0.11.2
+requests-oauthlib==1.3.0
+sentry-sdk==0.13.2
```

Comment:
As of [version 0.12.0](https://github.com/getsentry/sentry-python/releases/tag/0.12.0), Sentry requires `aiocontextvars` when using Channels and running on Python 3.5 or 3.6. (The release notes also mention that previously it leaked memory in these scenarios, so this may be the cause of some of the memory leaks noticed in production earlier today).

`sentry-sdk` does not automatically pull this in (my guess is they don't want to pull it in for projects that don't need it), so we will have to install it manually on versions < 3.7. I think this will work (untested):
`aiocontextvars==0.2.2; python_version < "3.7"`

Reply:
Ah I missed that when I was skimming through change logs.